package.json

```diff
@@ -11,6 +11,10 @@
     "node": ">=5"
   },
   "scripts": {
+    "prepublish": "npm run install-cli && npm run install-extension && npm run install-viewer",
```

Comment:
ah, very interesting. SG. I hadn't seen that change, only the controversy back in 2013 when it started to run `prepublish` on `install` :) We should hunt down the current discussions on this